src/OpenTelemetry/Metrics/MetricReaderOptions.cs

```diff
@@ -27,7 +27,7 @@ public class MetricReaderOptions
     public MetricReaderTemporalityPreference TemporalityPreference { get; set; } = MetricReaderTemporalityPreference.Cumulative;
 
     /// <summary>
-    /// Gets the <see cref="PeriodicExportingMetricReaderOptions" /> options.
+    /// Gets or sets the <see cref="PeriodicExportingMetricReaderOptions" />.
     /// </summary>
-    public PeriodicExportingMetricReaderOptions PeriodicExportingMetricReaderOptions { get; private set; } = new PeriodicExportingMetricReaderOptions();
+    public PeriodicExportingMetricReaderOptions PeriodicExportingMetricReaderOptions { get; set; } = new PeriodicExportingMetricReaderOptions();
```

Comment:
@alanwest This probably introduces the chance of a null reference exception we need to guard against where we access the value. Alternatively, don't use an auto-prop here so you can do the guard in the setter if it is being set to `null`.

Probably a rare thing we can do in a follow-up.

Reply:
🤦 yep, good catch. Pushed a fix, could add a test to in a sec.

Reply:
> Alternatively, don't use an auto-prop here so you can do the guard in the setter if it is being set to null.

Maybe this is a good pattern to consider across all our options classes that have nested settings. Could do this in the future.